File: RepoTransBench/target_projects/Python/C++/kevinburke_hamms/public_tests/test_public_hamms_server.cpp
#include <gtest/gtest.h>
#include <map>
#include <string>

TEST(PublicHammsServer, TrueAssertion) {
    ASSERT_EQ(100 / 5, 20);
}

TEST(PublicHammsServer, MapKeyPresence) {
    std::map<std::string, int> config = {{"foo", 5}, {"bar", 9}};
    ASSERT_NE(config.find("bar"), config.end());
}

File: RepoTransBench/target_projects/Python/Rust/kevinburke_hamms/public_tests/test_public_hamms_server.rs
#[test]
fn test_public_server_true() {
    assert_eq!(100 / 5, 20);
}

#[test]
fn test_public_server_other() {
    use std::collections::HashMap;
    let mut config = HashMap::new();
    config.insert("foo", 5);
    config.insert("bar", 9);
    assert!(config.contains_key("bar"));
}